scripts/zones/Qufim_Island/IDs.lua

```diff
@@ -64,6 +64,7 @@ zones[xi.zone.QUFIM_ISLAND] =
             [17293536] = 17293537, -- -137.651 -23.507 231.528
         },
         OPHIOTAURUS = GetFirstID('Ophiotaurus'),
+        ATKORKAMUY  = GetFirstID('Atkorkamuy'),
```

Comment:
I don't see where this is being used?

Reply:
I think I initially put that in for if I needed it but didn't end up needing it, and left it for basically completeness sake since he is part of the zone still. Can remove if you want

Reply:
yeah kill it. we only add these for when we need to get the objects id, and as a timed spawn nothings doing that right now

Reply:
It had me merge in fixes that should already be in the branch *again* though, and no clue why it keeps doing that. But still only the 6 files changed for the PR, I have no clue why my branch keeps acting up with the stuff from base >_<

Reply:
Stop pulling after you rebase, this is user error. Use force push instead